src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -5,9 +5,10 @@
   cursor: pointer;
   .workItemList_workItemType {
     span{
-      font-size: $font12;
+      font-size: $font14;
```

Comment:
check with @bdellasc before changing font sizes, as the UXD Visuals are based off of particular font sizes. If the size is too small, then he should know.